tests/test_orm.py

```diff
@@ -76,15 +77,15 @@ def test_make_and_populate_new_db(tmp_path: str) -> None:
         system=uname.system,  # Operating system
         program="guestimate",
         version="v0.1.2beta3",
-        fragsize=17,
+        fragsize=100,
         kmersize=17,
     )
     # Test the __repr__
     assert repr(config) == (
         "Configuration(configuration_id=None,"
```

Comment:
If using fixtures, we could have a database containing the various dummy results as a fixture, and assert that the responses from the newly-created database are the same as those in the fixture database.

We could have one run per test group in the fixture db.

What do you think?

Reply:
I think a large test database of (mock or at least realistic) numbers would be good as a fixture. Ideally something useful for the plot tests too.